### DISCUSSION TITLE
Filtering search results

### DISCUSSION THREAD

**Discussion by michallauko** [dailydotdev/daily#863] [Feature Requests]
Would it be possible to filter search results? Now I am getting mixed articles by date. It would be great to search for a term and see articles ordered from the newest ones, or at least have such an option.

**Comment by idoshamun** [comment]
Hi @michallauko,

Unfortunately now it's not possible but we have an active discussion on it. So let's keep it there
https://github.com/dailydotdev/daily/discussions/294
